code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/QuestionContent.java

```diff
@@ -0,0 +1,59 @@
+package com.cmput301w21t06.crowdfly.Controllers;
+

```

Comment:
small note, for any adapter classes we have went with the naming convention of (ClassName)Adapter, ex. we changed ExperimentContent to ExperimentAdapter

Reply:
Weird, that file should be outdated and deleted from my end... I guess Git decided to keep the file for some reason